challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -5172,36 +5335,50 @@
       ],
       "tests": [
         {
-          "text": "'<code>jortsort</code> should be a function.'",
-          "testString": "assert(typeof jortsort=='function','<code>jortsort</code> should be a function.');"
+          "text": "<code>jortsort</code> should be a function.",
+          "testString":
+            "assert(typeof jortsort=='function','<code>jortsort</code> should be a function.');"
         },
         {
-          "text": "'<code>jortsort('+JSON.stringify(tests[0])+')</code> should return a boolean.'",
-          "testString": "assert(typeof jortsort(tests[0].slice())=='boolean','<code>jortsort('+JSON.stringify(tests[0])+')</code> should return a boolean.');"
+          "text": "<code>jortsort([1,2,3,4,5])</code> should return a boolean.",
+          "testString":
+            "assert(typeof jortsort([1,2,3,4,5].slice())=='boolean','<code>jortsort([1,2,3,4,5])</code> should return a boolean.');"
```

Comment:
`slice()` is not needed anymore as the array used is not stored in a variable. It was used earlier as the function would mutate the tests.

Reply:
Okay! I'll remove that and update the PR.